indico/modules/designer/templates/_list.html

```diff
@@ -60,31 +58,55 @@
+                                title="{% trans %}Toggle ticket template for this category{% endtrans %}"
+                                data-href="{{ url_for('.toggle_category_default', target, template_id=tpl.id) }}"
+                                data-method="POST"
+                                data-update="#template-list"
+                            {% endif %}>
+                        </a>
+                    {% endif %}
+                        {% set clonable = tpl.is_clonable or target == tpl.category.get_root() %}
+                        <button class="i-button icon-copy icon-only text-color borderless hide-if-locked
+                            {{ 'not-clonable' if not clonable }}"
+                            {% if not clonable %}
+                                title="{% trans %}This template cannot be cloned{% endtrans %}"
+                            {% else %}
                                 title="{% trans %}Clone this template{% endtrans %}"
                                 data-href="{{ url_for('.clone_template', target, template_id=tpl.id) }}"
                                 data-method="POST"
                                 data-update="#template-list"
                                 data-confirm="{% trans title=tpl.title -%}
                                                   Are you sure you want to clone template '{{ title }}''?
                                               {%- endtrans %}">
+                            {% endif %}
                         </button>
-                    {% endif %}
                     {% if not inherited %}
                         <a class="i-button icon-edit icon-only text-color borderless hide-if-locked"
                            href="{{ url_for('.edit_template', tpl) }}"
                            title="{% trans %}Edit template{% endtrans %}"></a>
                     {% endif %}
                     {% if not inherited %}
-                        <button class="i-button icon-remove icon-only text-color borderless danger hide-if-locked"
-                                title="{% trans %}Delete tempalte{% endtrans %}"
-                                data-href="{{ url_for('.delete_template', tpl) }}"
-                                data-method="DELETE"
-                                data-update="#template-list"
-                                data-confirm="{% trans title=tpl.title -%}
-                                                  Are you sure you want to delete template '{{ title }}'?
-                                              {%- endtrans %}">
+                        <button class="i-button icon-remove icon-only text-color borderless danger hide-if-locked
+                                {{ 'not-deletable' if not tpl.can_be_deleted }}"
+                                {% if not tpl.can_be_deleted %}
+                                    title="{% trans %}This template cannot be deleted because it's used by another registration form{% endtrans %}"
```

Comment:
"it is" instead of "it's" (the shortcuts are very informal and not recommended in most written communication, chats etc. aside)